mmcv/transforms/utils.py

```diff
@@ -81,13 +81,13 @@ def __get__(self, obj, cls):
 
 @contextmanager
 def cache_random_params(transforms: Union[BaseTransform, Iterable]):
-    """Context-manager that enables the cache of cacheable methods in
-    transforms.
+    """Context-manager that enables the cache of return values of methods
+    decorated by `cache_randomness` in transforms.
```

Comment:
```suggestion
    decorated by ``cache_randomness`` in transforms.
```
Double backquotes for code samples